styles/components/_misc.scss

```diff
@@ -39,3 +39,26 @@
   background-color: #000;
   border-radius: 4px;
 }
+
+.ratingStrip {
+  .ratingIcons {
+    display: inline-flex;
+    flex-direction: row-reverse;
```

Comment:
Wish we had a previous sibling selector in CSS. Instead, I'm fudging it with reversing the order in flex.